modules/ip6/datapath/ip6_forward.c

```diff
@@ -35,22 +35,20 @@ ip6_forward_process(struct rte_graph *graph, struct rte_node *node, void **objs,
 	return nb_objs;
 }
 
-static struct rte_node_register forward_node = {
+static struct rte_node_register node = {
```

Comment:
Why rename this variable here?

Reply:
Bad rebase, I'll move that in the proper commit.